server/views/applications/pages/previous-unspent-convictions/unspent-convictions.njk

```diff
@@ -27,32 +27,7 @@
               ]
             }
           },
-          rows: [
-            {
-              key: {
-                text: 'Number of convictions of the same type'
-              },
-              value: {
-                text: unspentConviction.numberOfConvictions
-              }
-            },
-            {
-              key: {
-                text: 'Are they currently serving a sentence for these convictions?'
-              },
-              value: {
-                text: unspentConviction.currentlyServing
-              }
-            },
-            {
-              key: {
-                text: 'Are there any safeguarding details to add about these convictions?'
-              },
-              value: {
-                text: unspentConviction.safeguarding
-              }
-            }
-          ]
+          rows: unspentConvictionsCardRows(unspentConviction)
```

Comment:
As mentioned earlier. this approach is nicer than the previous method, but I'd suggest calling this function in the controller and passing the `cardRows` into the render.

Reply:
actually, given that this is nested within a for loop, i think I'll leave it as is for now.